datacenter/ucp/2.2/guides/get-support.md

```diff
@@ -36,7 +36,7 @@ docker container run --rm \
   --name ucp \
   -v /var/run/docker.sock:/var/run/docker.sock \
   {{ page.ucp_org }}/{{ page.ucp_repo }}:{{ page.ucp_version }} \
-  support > docker-support.tgz
+  support --log-driver none > docker-support.tgz
```

Comment:
The `--log-driver none` arg has to come before the image; it's an arg for `docker run`, not for the support command.